bin/Runner.ts

```diff
@@ -28,39 +28,45 @@ Options:
   -e    always exit with status code 0 on test errors
   -t    regex for test IRIs to run
   -i    JSON string with custom options that need to be passed to the engine
+  -d    Time out duration for test cases (in milliseconds, default 3000)
 `);
   process.exit(1);
 }
 
-// Set format
-let format = 'detailed';
-if (args.o) {
-  format = args.o;
-}
-
-// Scope to a specification
-const specification = args.s;
-
-// Optional test IRI regex
-const testRegex = new RegExp(args.t);
-
 // Enable caching if needed
-let cachePath: string = null;
+// tslint:disable-next-line: variable-name
+let _cachePath: string = null;
 if (args.c) {
-  cachePath = Path.join(process.cwd(), (args.c === true ? '.rdf-test-suite-cache/' : args.c));
```

Comment:
Any reason for changing this? If not, I'd keep it as `cachePath`.

Reply:
To signal this value should actually not be used anymore, and if you do use it, you have to be sure of what you are doing. This logic should actually be handled by the config and it's related code, but it required a little bit too much changes to do it cleanly.

Reply:
`_` is typically used to indicate private fields in classes, so it's a bit confusing to use it for your reason here.
Also, `cachePath` never leaves the scope of this file, so I don't see any problem with keeping `cachePath`.